XCUITests/ReaderViewUITest.swift

```diff
@@ -31,7 +30,6 @@ class ReaderViewTest: BaseTestCase {
         waitUntilPageLoad()
         waitForExistence(app.buttons["Reader View"], timeout: 5)
         app.buttons["Reader View"].tap()
-        app.buttons["Reload"].tap()
```

Comment:
@isabelrios given the home button replacing reload, would this be reasonable to do? Is there a reason why reload is needed before adding an item to a reading list?